src/ci/docker/x86_64-gnu-llvm-3.7/Dockerfile

```diff
@@ -26,5 +26,6 @@ ENTRYPOINT ["/usr/bin/dumb-init", "--"]
 
 ENV RUST_CONFIGURE_ARGS \
       --build=x86_64-unknown-linux-gnu \
-      --llvm-root=/usr/lib/llvm-3.7
+      --llvm-root=/usr/lib/llvm-3.7 \
+      --enable-test-miri
```

Comment:
On this travis instance it ran from `[01:09:24]` to `[01:16:11]`, so it needs around 7 minutes.

This is weird. It should be sharing a few libraries with rustc. But it seems miri is the only thing built in stage2. It should be perfectly possible to build miri in stage1. I'll investigate.